README.md

```diff
@@ -1,2 +1,152 @@
-# International-Business-School
-HTML/CSS &amp; JavaScript capstone project
+Hello, I´m Marco Díaz
+
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🔭 Video Presentation](#video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [HTML/CSS & JavaScript capstone project] <a name="about-project"></a>
+
+**[Learning objectives]**
+
+In this capstone project I am going to build is based on an online website for a conference. Microverse provide some design guidelines for the website.
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS & JAVASCRIPT
+
+GitHub & Visual Studio Code
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Navbar
+- Mobile responsive
+- DOM Manipulation
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://marcodiaz2000.github.io/International-Business-School/)
```

Comment:
Nice job deploying your project with Github pages👍 currently, the link to your About page on your deployed project is not working properly🥲 Kindly fix the issue as this can lead to a frustrating user experience👍
<img width="701" alt="image" src="https://user-images.githubusercontent.com/64643995/233147021-3a4c00d5-0cda-452e-8ca4-7310a727828a.png">

When your logo link is clicked, it displays the image below🥲 I suggest it should lead to your home page👍 as this makes the experience of the user even better👍
<img width="676" alt="image" src="https://user-images.githubusercontent.com/64643995/233150106-93ab56fa-f8fb-4d21-a836-27672b4cd33c.png">